beta/src/pages/learn/index.md

```diff
@@ -4,716 +4,491 @@ title: Quick Start
+JSX is stricter than HTML. You have to close tags like `<br />`. Your component also can't return multiple JSX tags. You have to wrap them into a shared parent, like a `<div>...</div>` or an empty `<>...</>` wrapper:
 
-To save you time, we provide **a brief overview of each chapter** below.
+```js {3,6}
+function AboutPage() {
+  return (
+    <>
+      <h1>About</h1>
+      <p>Hello there.<br />How do you do?</p>
+    </>
+  );
+}
+```
 
-### Chapter 1 overview: Describing the UI {/*chapter-1-overview-describing-the-ui*/}
+If you have a lot of HTML to port to JSX, you can use an [online converter](https://transform.tools/html-to-jsx).
 
-React applications are built from isolated pieces of UI called ["components"](/learn/your-first-component). A React component is a JavaScript function that you can sprinkle with markup. Components can be as small as a button, or as large as an entire page. Here, a *parent* `Gallery` component renders three *child* `Profile` components:
+## Adding styles {/*adding-styles*/}
 
-<Sandpack>
+In React, you specify a CSS class with `className`. It works the same way as HTML [`class`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/class) attribute:
 
 ```js
-function Profile() {
-  return (
-    <img
-      src="https://i.imgur.com/MK3eW3As.jpg"
-      alt="Katherine Johnson"
-      className="avatar"
-    />
-  );
-}
+<img className="avatar" />
+```
 
-export default function Gallery() {
-  return (
-    <section>
-      <h1>Amazing scientists</h1>
-      <Profile />
-      <Profile />
-      <Profile />
-    </section>
-  );
+Then you write the CSS rules for it in a separate CSS file:
+
+```css
+/* In your CSS */
+.avatar {
+  border-radius: 50%;
 }
 ```
 
-```css
-img { margin: 0 10px 10px 0; height: 90px; }
-.avatar { border-radius: 50%; }
+React does not prescribe how you add CSS files. In the simplest case, you'll add a [`<link>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link) tag to your HTML. If you use a build tool or a framework, consult its documentation to learn how to add a CSS file to your project.
+
+## Displaying data {/*displaying-data*/}
+
+JSX lets you put markup into JavaScript. Curly braces let you "escape back" into JavaScript so that you can embed some variable from your code and display it to the user. For example, this will display `user.name`:
+
+```js {3}
+return (
+  <h1>
+    {user.name}
+  </h1>
+);
 ```
 
-</Sandpack>
+You can also "escape into JavaScript" from JSX attributes, but you have to use curlies *instead of* quotes. For example, `className="avatar"` passes the `"avatar"` string as the CSS class, but `src={user.imageUrl}` reads the JavaScript `user.imageUrl` variable value, and then passes that value as the `src` attribute:
```

Comment:
**problem**: "curlies" is cute but may be confusing for non-English speakers

**possible solution**: 
- add the curly symbols, like so: "but you have to use curlies (`{` and `}`)" or: "but you have to use curlies (`{}`)"